torch/utils/data/dataloader.py

```diff
@@ -6,6 +6,7 @@
 import traceback
 import threading
 from torch._six import string_classes
+import numpy as np
```

Comment:
I thought we tried to avoid importing numpy in pytorch vode because it's not a strict dependency.  Did we relax that?

Reply:
No, it should be optional! Nice catch, thank you!